elliott

```diff
@@ -335,10 +336,14 @@ advisory.
               metavar='ADVISORY_TYPE',
               type=click.Choice(['image', 'rpm', 'security']),
               help="Use the default value from ocp-build-data for ADVISORY_TYPE [image, rpm, security]")
-@click.option("--auto",
-              required=False,
-              default=False, is_flag=True,
-              help="Auto-find mode, adds bugs based on --group")
+@click.option("--mode",
+              required=True,
+              type=click.Choice(['LIST', 'SWEEP', 'PAYLOAD']),
```

Comment:
Style comments: For other multiple-choice stuff we usually use all lower case. Also we don't have many other help commands going over several lines like that. Maybe best to include the specifics in the larger paragraph doctext?

Reply:
Using mode is cool though. Getting rid of `--auto` will be more intuitive I think.

Reply:
What if `list` was the default? I think I'd like that.